package.json

```diff
@@ -134,6 +134,15 @@
             "maxNumberOfProblems": 100,
             "trace.server": "off"
           }
+        },
+        "terraform.rootModules": {
```

Comment:
I guess what we call the option here is orthogonal to how it is sent to the server.

The server doesn't support explicit `workspace/configuration` calls yet so doesn't matter as much, but it will at some point and so it will be asking for an explicit section of a config in the future.

The convention I saw in gopls is that the server asks for sections called `gopls` and `gopls-workspaceName`. I'm not saying we have to care about workspace scoping yet, but I do think we should call the section `terraform-ls` if possible, or somehow separate what's server settings and what's client settings so it's not tempting to mix the two.

Reply:
Good point, I think using `terraform-ls` would be best so it's not confusing for people who do choose to use a different language server.